function foo() {
  try {

  } catch (undefined) {
    console.log(error);
  }
}
